data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -281,36 +272,36 @@ public void testFullScan() {
     expected.addAll(file3FirstSnapshotRecords);
 
     Set<Record> records = Sets.newHashSet(results);
-    Assert.assertEquals(
-        "Should produce correct number of records", expected.size(), records.size());
-    Assert.assertEquals("Random record set should match", Sets.newHashSet(expected), records);
+    Assertions.assertThat(expected)
+        .hasSize(records.size())
+        .as("Should produce correct number of records");
+    Assertions.assertThat(records)
+        .isEqualTo(Sets.newHashSet(expected))
+        .as("Random record set should match");
```

Comment:
same as above. Please also make sure this is fixed in all the other places in this PR